app/crud.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy.orm import Session
+from . import models, schemas
+
+def create_city(db: Session, city: schemas.CityCreate):
+    db_city = models.City(name=city.name, additional_info=city.additional_info)
+    db.add(db_city)
+    db.commit()
+    db.refresh(db_city)
+    return db_city
+
+def get_city(db: Session, city_id: int):
```

Comment:
It might be beneficial to handle the case where the city is not found, possibly by raising an exception or returning a specific error message.